.circleci/config.yml

```diff
@@ -182,6 +182,24 @@ commands:
                     make sdkRegistryUpload
                   fi
 
+  publish-sdk-registry-named-snapshot:
+    steps:
+      - unless:
+          condition: << pipeline.parameters.mapbox_upstream >>
+          steps:
+            - run:
+                name: Publish the SDK snapshot based on the keyword in commit message
+                command: |
+                  COMMIT_SHA=$(git rev-parse --short HEAD)
+                  LAST_RELEASE=$(git describe --tags --abbrev=0)
+                  if [[ -n ${CIRCLE_BRANCH+x} ]] && [[ $CIRCLE_BRANCH != "main" ]] && [[ "$(git log -1)" = *"publish_android_snapshot"* ]]; then
+                    BRANCH_ID=$(echo $CIRCLE_BRANCH)
```

Comment:
I think `BRANCH_ID` can be removed and postfix removal`${BRANCH_ID% -}` is not needed, since `CIRCLE_BRANCH` should contain just a branch name.

`sed -i -e "s/^VERSION_NAME=.*/VERSION_NAME=${LAST_RELEASE:9}-${CIRCLE_BRANCH}-SNAPSHOT/" gradle.properties` wdyt?